Sources/SafariWalletCore/Client/Unmarshal/UnmarshalModels.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  UnmarshalModels.swift
+//  
+//
+//  Created by Tassilo von Gerlach on 11/8/21.
+//
+
+import Foundation
+import MEWwalletKit
+
+public enum Unmarshal {
+    
+    public struct TokenTransactionsResponse: Codable {
```

Comment:
If the codable models should be accessible by the wallet then we'll need to make all the params public also.